src/pages/debrief/cat-a-mod2/components/safety-and-balance-card/__tests__/safety-and-balance-card.cat-a-mod2.spec.ts

```diff
@@ -55,8 +55,8 @@ describe('SafetyAndBalanceCardCatAMod2Component', () => {
   beforeEach(async(() => {
     fixture = TestBed.createComponent(SafetyAndBalanceCardCatAMod2Component);
     store$ = TestBed.get(Store);
-    store$.dispatch(new StartTest(105, TestCategory.EUA2M2));
-    store$.dispatch(new PopulateTestCategory(TestCategory.EUA2M2));
+    store$.dispatch(new StartTest(105, TestCategory.EUAM2));
```

Comment:
Any reason for changing this from using 'EUA2M2' to 'EUAM2'?

Reply:
wanted to see if it would affect the component loading up to see if that had any affect on the HTML produced which was causing me issues, nothing permanent, will change back before merge and final PR